Bugzilla/Config/General.pm

```diff
@@ -33,8 +33,9 @@ use constant get_param_list => (
 
     {
         name    => 'utf8',
-        type    => 'b',
-        default => '0',
+        type    => 't',
+        choices => [ '1', 'utf8', 'utf8mb4' ],
+        default => 'utf8',
```

Comment:
Should be instead:

type => 's',
choices => [ 'utf8', 'utf8mb4' ],
default => 'utf8',